### PR TITLE
Updated to support Roslyn version 2.6.1

### PR DESCRIPTION
**Pull request by anglodutch**
#178 This functionally works to support version 2.6.1 of Roslyn although there may be a better way of resolving the problems with FixAllState.Create being removed by MS.

### CODE REVIEW

File: build/common.props [hunk 1]
```diff
@@ -3,9 +3,9 @@
 		<Features>strict</Features>
 		<Deterministic>True</Deterministic>
 		<TreatWarningsAsErrors>True</TreatWarningsAsErrors>
-		<EditorPackageVersion>1.0.4</EditorPackageVersion>
-		<RoslynPackageVersion>2.4.0</RoslynPackageVersion>
-		<RoslynAssemblyVersion>2.4.0.0</RoslynAssemblyVersion>
+		<EditorPackageVersion>1.0.5</EditorPackageVersion>
+		<RoslynPackageVersion>2.6.0</RoslynPackageVersion>
```
Comment (aelij) [inline build/common.props]: Should be 2.6.1

File: src/RoslynPad.Roslyn/CodeFixes/CodeFixService.cs [hunk 2]
```diff
@@ -355,6 +356,33 @@ await diagnosticsWithSameSpan.OrderByDescending(d => d.Severity)
                 result.Add(codeFix);
             }
 
+            private FixAllState CreateFixAllState(
```
Comment (aelij) [inline src/RoslynPad.Roslyn/CodeFixes/CodeFixService.cs]: Did you just take the fix all provider? Perhaps it's a good idea to copy the whole `CodeFixService` class again.

File: build/common.props (hunk 1 above)
Comment (anglodutch) [reply build/common.props]: Corected

File: src/RoslynPad.Roslyn/CodeFixes/CodeFixService.cs (hunk 2 above)
Comment (anglodutch) [reply src/RoslynPad.Roslyn/CodeFixes/CodeFixService.cs]: I wan't aware this was a copy of a Roslyn class but now I've looked I made the same changes that were made in the commit as the was made to Roslyn when they moved the FixAllDiagnosticProvider class. Comparing the two there are some differences with the Roslyn version but I'm not 100% sure what should be merged. I assume you've made some changes to it after copying it, or is it just a straight copy that you have updated every so often?

Comment (anglodutch) [reply src/RoslynPad.Roslyn/CodeFixes/CodeFixService.cs]: @aelij Hi, did you see my comment above?

Comment (aelij) [reply src/RoslynPad.Roslyn/CodeFixes/CodeFixService.cs]: I did make some minor changes, I just don't remember exactly what. Let's leave it as-is for now. Did you check that code fixes work?

Comment (anglodutch) [reply src/RoslynPad.Roslyn/CodeFixes/CodeFixService.cs]: Yes, at the moment we're using these changes a part of our project as we need the 2.6.1 support. The code is also a straight copy from Roslyn.

**Pull request merged**